### PR TITLE
update to pyproject

### PR DESCRIPTION
**Pull request by knaaptime**

### CODE REVIEW

File: esda/__init__.py
Comment (martinfleis) [inline esda/__init__.py:1]: `__version__` retrieval needs to be updated here to work with setuptools_scm.

```py
import contextlib
from importlib.metadata import PackageNotFoundError, version

with contextlib.suppress(PackageNotFoundError):
    __version__ = version("esda")

Comment (knaaptime) [reply esda/__init__.py:1]: doh. thanks! I did it right the first time, but then realized i'd done it on top of the wrong branch and missed this piece the second go 'round

File: esda/__init__.py [hunk 1]
```diff
@@ -3,6 +3,10 @@
 =================================================
 
 """
+
+import contextlib
+from importlib.metadata import PackageNotFoundError, version
+
 from . import _version, adbscan, shape  # noqa F401
```
Comment (martinfleis) [inline esda/__init__.py:10]:
```suggestion
from . import adbscan, shape  # noqa F401
```

this probably should not be there?

Comment (jGaboardi) [reply esda/__init__.py:10]: @knaaptime Did you see Martin's comment here?

**Review by jGaboardi — CHANGES_REQUESTED** [review]
All looks well after addressing @martinfleis's comment

**Comment by knaaptime** [comment]
ok i think i got the last piece

**Review by sjsrey — APPROVED** [review]

**Pull request merged**